src/c++/perf_analyzer/genai-perf/README.md

```diff
@@ -407,7 +407,9 @@ The HuggingFace dataset to use for prompts.
 
 ##### `--input-file <path>`
 
-The input file containing the single prompt to use for profiling.
+The input file containing the prompts to use for profiling.
+Each line should be a JSON object with a 'text_input' field in JSONL format.
```

Comment:
Maybe link to JSONL documentation?

https://jsonlines.org

Reply:
Thanks for reviewing and making this suggestion! Vinh said that JSONL is ubiquitous in the LLM space (pinged you a link to that conversation), so I'd rather skip this link for three reasons:
1. We don't do it for other filetypes (e.g. JSON, XML).
2. This would be a decentralized way to do it. If we want to link to documentation for each filetype, we should do it somewhere central to Triton rather than in some references.
3. We would need to maintain an external link (with some responsibility for it not breaking and its content).

I agree this would be a good idea if we were using an unknown filetype, which is why I wanted to avoid using that type of filetype. :) Do you think we're good to not add a link given JSONL's popularity?